AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -79,53 +80,65 @@ The Alembic exporter can be customized by setting the properties on this compone
-|__Detailed Log__ |{TODO}|
-|__Begin Recording__ |{TODO}|
+|__Scale Factor__ |Choose scale factor to convert system units, for example 0.1, converts it to 1/10 size. This also affects position and speed.|
+|__Capture Scope__<br/>"Entire Scene",<br/>"Target Branch" | Choose the scope of the export. By default the entire scene will be exported but it can be configured to export just a branch (or hierarchy) of the scene. ![Alembic Export Target](images/abc_export_target.png) |
+|__Assume Non Skinned Meshes Are Constant__ | If enabled, will not capture animation on static meshes |
+|__Capture MeshRenderer__ |Enable to record MeshRenderer components|
+|__Capture SkinnedMeshRenderer__ |Enable to record SkinnedMeshRenderer components|
+|__Capture Particle__ |Enable to record ParticleSystem components|
+|__Capture Camera__ |Enable to record Camera components|
+|__Mesh Components Normals__ |Enable to record mesh normals|
+|__Mesh Components UV1__ |Enable to record mesh UV1|
+|__Mesh Components UV2__ |Enable to record mesh UV2|
+|__Mesh Components Vertex Color__ |Enable to record vertex colors|
+|__Mesh Components Submeshes__ |Enable to record submeshes|
+|__Capture On Start__ |Begin capturing in Start() when scene with AlembicExporter component is loaded|
+|__Ignore First Frame__ |If enabled, do not capture first frame. Only available if **Capture On Start** enabled|
+|__Max Capture Frame__ |The frame to stop capturing at|
+|__Detailed Log__ |Provide detailed Debug logs of each frame that is captured|
+|__Begin Recording__ |In play mode, click button to begin recording. Use if **Capture On Start** is disabled|
 |__One Shot__|Button to export the current frame to the ABC file.|
 
+> ***Note:*** The current capture target object is determined at the start of capture, and does not change in the middle. Enabling or disabling the object does not affect the capture.
+> Also, any objects generated after the start of the capture are not captured.
+
+> ***Note:*** Be careful when deleting the target object in the middle of capture. In this case, the capture of that object will be interrupted, and the Alembic file may have a non-uniform number of samples as a result. Some software may not handle this properly.
+
+> ***Note:*** An ID (e.g. "(0000283C)") is given to the name of the Alembic node on export. This is a measure to avoid name conflicts as Alembic has a rule that each node in a hierarchy should have a unique name. 
+
+
+## Known Issues
+
+* Material export is currently not supported
+
 # Controlling Alembic playback
 
 The import and playback of Alembic data is controlled by the `Alembic Stream Player` component.
 
-If you change the Time parameter you can see that Mesh animate. To play the animation this parameter can be controlled from the Timeline, Animator component or via scripts.
+If you change the Time parameter you can see the Mesh animate. To play the animation this parameter can be controlled from the Timeline, Animator component or via scripts.
```

Comment:
I would replace Mesh with Objects (to cover particles and whatever else gets added in the future).

The Controlling Alembic playback section is a bit redundant with what was explained before. We can leave that to Amy to figure out

Reply:
I think the first one is like a TL;DR, and this is a more detailed version